pkg/webconsoleurl/lookup.go

```diff
@@ -0,0 +1,35 @@
+package webconsoleurl
```

Comment:
can we move to this pkg/util/webconsole/url_lookup.go ?

Reply:
Done.